companies/koch-media-gmbh.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "koch-media-gmbh",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "entertainment",
+        "commerce"
+    ],
+    "name": "Koch Media GmbH",
+    "runs": [
+        "Deep Silver"
+    ],
+    "address": "Lochhamer Str. 9\n82152 Planegg\nGermany",
+    "phone": "+49 89 24245405",
+    "fax": "+49 89 242453405",
+    "web": "https://www.kochmedia.com",
+    "sources": [
+        "https://www.deepsilver.com/en/privacy-policy/",
+        "https://www.kochmedia.com/en/privacy-policy/"
+    ],
+    "comments": [
+        "Check https://www.deepsilver.com/en/privacy-policy/ to obtain the email address of the DPO."
```

Comment:
:D 

We should at least specify the general email from https://www.kochmedia.com/en/imprint/ in the record. Usually, we'd leave it at that but if you want to keep the comment, that's okay.

Reply:
Okay. 

Lets keep the comment. :D